### DISCUSSION TITLE
Cannot read properties of undefined (reading 'createLanguageService')"

### DISCUSSION THREAD

**Discussion by RizaHKhan** [LunarVim/LunarVim#3192] [General]
Hello, 

I'm seeing this error every-time I open a `.vue` file:
![image](https://user-images.githubusercontent.com/51720003/194781501-2d45e49c-0948-4e4d-896c-072a35c9a344.png)

Can someone help me debug? Really bad with the whole LSP and how it works :( 

Thank you!

**Comment by kenvlo** [comment]
Try downgrading to Volar 0.4 (https://github.com/johnsoncodehk/volar/releases/tag/v0.40.13).

**Comment by MrWazaby** [comment]
Hello, I was able to fix this after reading [this issue in the volar repository](https://github.com/johnsoncodehk/volar/issues/1973#issuecomment-1274022271).

So I edited `~/.local/share/lunarvim/lvim/snapshot/default.json` and changed the value of `mason-lspconfig.nvim` :  
```
"mason-lspconfig.nvim": {
    "commit": "0eb7cfe"
  },
```

Then, run `:LvimSyncCorePlugins` and it should be ok. I hope this help, but we may need to open an issue to ask LunarVim maintainers to update the `mason-lspconfig.nvim` pinned version.

**Reply by RizaHKhan** [reply]
This fixed my issue. Thank you @MrWazaby
